assets/sass/patterns/molecules/teaser.scss

```diff
@@ -16,57 +16,84 @@
 .teaser__context_label_list {
   @include nospace();
   position: relative;
-  @include truncate-with-ellipsis();
-
+  display: inline;
   .listing-list--highlights & {
     display: none;
   }
 
 }
 
+.teaser__curation_label_list {
+  @extend .teaser__context_label_list;
+}
+
 .teaser__context_label_item {
-  display: inline;
+  display: inline-block;
   list-style-type: none;
-  @include nospace();
-
-  & .teaser__context_label:after {
-    content: ", ";
-  }
-
-  &:last-child {
-    & .teaser__context_label:after {
-      content: "";
-    }
-  }
+  //@include nospace();
+  margin-bottom: 0.75rem;
+
+  //& .teaser__context_label:after {
```

Comment:
If this code that is commented out and will no longer be used then it can be deleted

Reply:
The commented code is deleted.